psi-ms.obo

```diff
@@ -17521,16 +17527,18 @@ relationship: has_value_type xsd:double ! The allowed value-type for this CV ter
 
 [Term]
 id: MS:1002664
-name: interaction score derived from cross-linking
-def: "Parent term for interaction scores derived from cross-linking." [PSI:PI]
-is_a: MS:1002675 ! cross-linking result details
-relationship: has_regexp MS:1002665 ! regular expression for interaction scores derived from cross-linking
+name: interaction score derived from crosslinking
+def: "Parent term for interaction scores derived from crosslinking." [PSI:PI]
+synonym: "interaction score derived from crosslinking" EXACT []
+is_a: MS:1002675 ! crosslinking result details
+relationship: has_regexp MS:1002665 ! regular expression for interaction scores derived from crosslinking
 relationship: has_value_type xsd:string ! The allowed value-type for this CV term
 
 [Term]
 id: MS:1002665
-name: regular expression for interaction scores derived from cross-linking
+name: regular expression for interaction scores derived from crosslinking
 def: "((?<int_ID>[0-9]+)[.](?<SUFFIX>[a|b]):(?<POS>[0-9]+|null):(?<SCORE_OR_VALUE>[-+]?[0-9]+(?:[.][0-9]+)?(?:[Ee][-+]?[0-9]+)?):(?<PASS_THRESHOLD>true|false))|(?<SCORE_ONLY>[-+]?[0-9]+(?:[.][0-9]+)?(?:[Ee][-+]?[0-9]+)?)" [PSI:PI]
+synonym: "regular expression for interaction scores derived from crosslinking" EXACT []
```

Comment:
this is probably intended to have a hyphen in it to acknowledge the original name, as done with other terms?

Reply:
yep, think so